wagtail/contrib/redirects/views.py

```diff
@@ -36,66 +37,88 @@
 )
 from wagtail.log_actions import log
 
-permission_checker = PermissionPolicyChecker(permission_policy)
 
+class RedirectPageColumn(Column):
+    cell_template_name = "wagtailredirects/redirect_page.html"
```

Comment:
Could we use the `_cell` suffix for the template name? Also, since the redirect target may not be a page, maybe name this `RedirectToColumn` or `RedirectTargetColumn`, e.g.

```suggestion
class RedirectTargetColumn(Column):
    cell_template_name = "wagtailredirects/redirect_target_cell.html"
```